lib/yardstick/v2_client/user.rb

```diff
@@ -18,7 +24,32 @@ class User
                     :email_verified,
                     :deleted_at,
                     :updated_at,
-                    :address
+                    :address,
+                    :time_zone
+
+      attr_accessor :paths
+
+      def self.process_response(resp, extra = {})
+        attrs = super
+        attrs.merge!(:paths => Paths.new(attrs[:paths]))
+      end
+
+      def create_booking(grant_id, params = {})
+        grant = find_grant(grant_id)
+        params.merge!(token: token)
+
+        response = Grant.post(grant.paths.bookings, body: params)
+        Booking.from_api(response)
+      end
+
+      def find_grant(grant_id)
+        response = Grant.get("#{paths.grants}/#{grant_id}", body: {
+          token: token,
+          user_id: id,
+          id: grant_id
+        })
```

Comment:
Indent the right brace the same as the first position after the preceding left parenthesis.